specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/examples/GetWorkspaceManagedSqlServerSecurityAlertPolicy.json

```diff
@@ -0,0 +1,31 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "wsg-7398",
+    "workspaceName": "testWorkspace",
+    "securityAlertPolicyName": "Default",
+    "api-version": "2019-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/wsg-7398/providers/Microsoft.Synapse/workspaces/testWorkspace/securityAlertPolicies/Default",
+        "name": "Default",
+        "type": "Microsoft.Synapse/workspaces/securityAlertPolicies",
+        "properties": {
+          "state": "Disabled",
+          "emailAccountAdmins": true,
+          "emailAddresses": [
+            "test@microsoft.com;user@microsoft.com"
+          ],
+          "disabledAlerts": [
+            "Access_Anomaly"
+          ],
+          "retentionDays": 0,
+          "storageAccountAccessKey": "",
+          "storageEndpoint": "https://mystorage.blob.core.windows.net"
+        }
```

Comment:
I think it should include the creationTime, here in all other examples

Reply:
Creation Time is a part of Security Alert Policy schema. As far as example is considered its same as that of the Sql Server Security Alert policy examples.

Reply:
done